frameworks/digital-outcomes-and-specialists/questions/briefs/successCriteriaParticipants.yml

```diff
@@ -1,6 +1,6 @@
-name: Proposal criteria
+name: Technical competence: proposal criteria
 id: successCriteria
-question: Proposal criteria
+question: Technical competence: proposal criteria
```

Comment:
I think you want single quotes around these values because the `:` in them is tripping up the parser.